.env.local.example

```diff
@@ -1,7 +1,9 @@
 # Update these with your Supabase details from your project settings > API
-NEXT_PUBLIC_SUPABASE_URL=https://your-project.supabase.co
-NEXT_PUBLIC_SUPABASE_ANON_KEY=
-SUPABASE_SERVICE_ROLE_KEY=
+NEXT_PUBLIC_SUPABASE_URL=https://jxqgwmmjxrrknnegarmt.supabase.co
+
+NEXT_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoiYW5vbiIsImlhdCI6MTYzODI4ODI2OCwiZXhwIjoxOTUzODY0MjY4fQ.UwfQYhN5KAkIT6qCkVgds9hep1m7dcW7NJiVTCF3l4s
+
+SUPABASE_SERVICE_ROLE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoic2VydmljZV9yb2xlIiwiaWF0IjoxNjM4Mjg4MjY4LCJleHAiOjE5NTM4NjQyNjh9.zYLTMPGhBrik_CN9iowa8ltmvq-y4wsTYQn1LSS6z5s
```

Comment:
@Marlstergo please don't commit your secret key to git history. Please email support@supabase.io to get the key rolled!